run_tests.sh

```diff
@@ -15,9 +16,9 @@ while getopts "p:b:h" opt "${EXTRAS[@]}"; do
         h)
             echo -e "NMSDK test usage instructions:\n";
             echo "ARGUMENTS:";
-            echo "-b <blender path> (optional, defaults to '.../../../../blender.exe')";
-            echo -e "\tThe relative or absolue path to the blender executable.";
-            echo "-p <python path> (optional, defaults to '../../../python/bin/python.exe')";
+	    echo "-b <blender path> (optional, defaults to what opens your .blend files)"
+            echo -e "\tThe relative or absolute path to the blender executable.";
```

Comment:
whoops! I clearly can't spell!
Also, I think the whitespaces in the lines above and below this one messed up??